src/matrFacility.ttl

```diff
@@ -147,6 +147,38 @@ somaf:Mine rdf:type owl:Class ;
            rdfs:label "mine"@en .
 
 
+###  http://sweetontology.net/matrFacility/Adit
+somaf:Mine rdf:type owl:Class ;
```

Comment:
should be somaf:Adit

Reply:
That's not how we had this classified on our end, but I tend to agree. I was looking at the rest of this that comes from the USMIN source (with reference back to the AGI dictionary).